src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,39 @@
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%,-50%);
+  padding: 10px;
+  @extend %position-content;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: $width-calendar;
+  margin: auto;
+  gap: $gap-between-day;
```

Comment:
Remember about styles properties order - [use this](https://codeguide.academy/html-css.html#css-order) or [this link](https://9elements.com/css-rule-order/). Fix it everywhere